app/controllers/application_controller.rb

```diff
@@ -73,6 +75,11 @@ def after_sign_in_path_for( resource )
     user_profile_path( resource.username )
   end
 
+  # Track all actions with Ahoy
+  def track_ahoy_visit
+    ahoy.track 'Ran action', request.path_parameters
```

Comment:
Will this log all form submissions and their data? E.g. comment text, emails, etc. Consider if that's desirable, this could be recording a lot of undesirable data / sensitive PII / spam.

Reply:
Consider limiting to only GET requests.